### PR TITLE
Ft/views

### PR DESCRIPTION
**Pull request by Bateyjosue**
# FEATURE DESCRIPTION

## TASK
In this project, I  add the Views associated with the Models and Controllers I've already created for my Blog App.

### Cheers
# Happy Review

### CODE REVIEW

**Comment by Bateyjosue** [comment]
Hey @review, I build the project design in mobile, Please consider that while doing a review. I will make sure to fix and add the desktop version late as I'm running behind the deadline. thanks for understanding

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,24 @@
...
+    <%= render partial: "users/shared/user", locals: {user: @user} %>
+  <p class="bio">
+    <h6>Bio:</h6>
+    <span>
+      <%= @user.bio %>
+    </span>
+  </p>
+  <ul class="user-detail__posts">
+    <% @user.posts.each do |post| %>
+      <li class="user-detail__posts_info">
```
Comment (KDlamini) [inline app/views/users/show.html.erb]: Please use the `last_three_post` method from your model to make sure that the User show page renders the user's 3 most recent posts as per project requirements.

`<% @user.last_three_post.each |post| %>`

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @Bateyjosue 🙋‍♀️,  

Great work so far! 💯 
You have done a great job creating views for the Blog app. 🎊 
There are some issues that you still need to work on improving your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 

✔️  Users controller `#index` action renders all users to their corresponding Views index page
✔️  Uses `post.recent_comments` to render 5 most recent comments
✔️  No linter errors

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Bateyjosue 

Your project is complete! Great job for implementing the changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**